src/material/core/tokens/m2/mat/_expansion.scss

```diff
@@ -0,0 +1,90 @@
+@use 'sass:map';
+@use '../../token-utils';
+@use '../../../theming/theming';
+@use '../../../style/sass-utils';
+@use '../../../typography/typography-utils';
+
+// The prefix used to generate the fully qualified name for tokens in this file.
+$prefix: (mat, expansion);
+
+// Tokens that can't be configured through Angular Material's current theming API,
+// but may be in a future version of the theming API.
+@function get-unthemable-tokens() {
+  @return (
+    container-shape: 4px,
+  );
+}
+
+// Tokens that can be configured through Angular Material's color theming API.
+@function get-color-tokens($config) {
+  $foreground: map.get($config, foreground);
+  $background: map.get($config, background);
+
+  @return (
+    container-background-color: theming.get-color-from-palette($background, card),
+    container-text-color: theming.get-color-from-palette($foreground, text),
+    actions-divider-color: theming.get-color-from-palette($foreground, divider),
+    header-hover-state-layer-color: theming.get-color-from-palette($background, hover),
+    header-focus-state-layer-color: theming.get-color-from-palette($background, hover),
```

Comment:
ty for including state layer color. Focus states are important for accessibility.

[OPtional] If it's not a lot of work, I suggest adding state layer opacity. We don't need that now, but we will probably need it in the future. I'm okay with hard coding the state layer opacity as 0.12 in `get-unthemable-tokens`.

Reply:
For that we'll have to put the state layer in a separate element (right now it's just a `background-color`). I'd rather we address it separately, because I'm trying to keep the DOM changes to a minimum since moving the declarations into the base styles already causes a bunch of broken targets.